composer.json

```diff
@@ -33,9 +33,9 @@
             "package": {
                 "name": "danskernesdigitalebibliotek/dpl-design-system",
                 "type": "drupal-library",
-                "version": "v1.0.2",
+                "version": "v1.0.3",
                 "dist": {
-                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-design-system/releases/download/v1.0.2/dist.zip",
+                    "url": "https://github.com/danskernesdigitalebibliotek/dpl-design-system/releases/download/release-release%2F1/dist.zip",
```

Comment:
Consider if we should refer to a tag here. `release/1` seems to be a moving target.

Whatever we do please update version accordingly.

Reply:
👍